tests/config/bindings/tests.yml

```diff
@@ -1,13 +1,18 @@
-# the config map is passed in a metadata map to the Invoke request for output bindings
-componentType: output-binding
+componentType: bindings
 components:
   - component: redis
     operations: ["create", "operations"]
     config:
-      key: $((uuid))
+      output_key: $((uuid))
+      readBindingTimeout: 60
   - component: azure.blobstorage
     operations: ["create", "operations", "get"]
     config:
-      blobName: $((uuid))
+      output_blobName: $((uuid))
+      readBindingTimeout: 60
   - component: azure.storagequeues
+    operations: ["create", "operations", "read"]
+  - component: kafka
     operations: ["create", "operations"]
+    config:
+      readBindingTimeout: 60
```

Comment:
can remove this since kafka does not have read operation yet.

Reply:
Done.